project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+              - ASHRAE IECC Climate Zone 2004|4B
+              - ASHRAE IECC Climate Zone 2004|4C
+              - ASHRAE IECC Climate Zone 2004|5A
+              - ASHRAE IECC Climate Zone 2004|5B
+              - ASHRAE IECC Climate Zone 2004|6A
+              - ASHRAE IECC Climate Zone 2004|6B
+              - ASHRAE IECC Climate Zone 2004|7A
+              - ASHRAE IECC Climate Zone 2004|7B
+            - Geometry Attic Type|Vented Attic
+            - or:
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+              - Insulation Ceiling|R-19
+              - Insulation Ceiling|R-30
+              - Insulation Ceiling|R-38
+        costs:
+          - value: 0.0443 # $/(ft^2 * Delta R-value), from original REMDB xlsx for blown cellulose (0.0154+0.019), with 1.289 inflation factor
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value)
+          - value: 0.1992 # $/(ft^2), from original REMDB xlsx for blown cellulose (0.1505+0.004), with 1.289 inflation factor
+            multiplier: Floor Area, Attic (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #General air sealing - 30% total reduction in ACH50 for homes with greater than 10 ACH50
+      #30% is consistent with median reduction documented in https://doi.org/10.20357/B7FP4D 
+      - &infiltration_30pct_reduction
+        option: Infiltration Reduction|30%
+        apply_logic:
+          - or:
+            - Infiltration|50 ACH50
+            - Infiltration|40 ACH50
+            - Infiltration|30 ACH50
+            - Infiltration|25 ACH50
+            - Infiltration|20 ACH50
+            - Infiltration|18.5 ACH50
+            - Infiltration|15 ACH50
+            - Infiltration|11.25 ACH50
+        costs:
+          - value: 0.50 # $0.50/ft2 is from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # Interpolated between 20% and 40% reductions to get $0.44/ft2,
+                        # then used 1.14 multiplier to inflate from Jan 2019 to Mar 2022
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+  
+  - upgrade_name: Intermediate Envelope
+    #
+    options:
+    #Light Touch Envelope Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *infiltration_30pct_reduction
+      - &ducts_category1 #improving leakage a lot, and insulation
+        option: Ducts|10% Leakage, R-8
+        apply_logic:
+          or: 
+            - Ducts|30% Leakage, R-4
```

Comment:
What about `- Ducts|30% Leakage, Uninsulated`?

Reply:
add 30% Leakage, Uninsulated in ducts_category1